pysodium/__init__.py

```diff
@@ -221,6 +221,13 @@ def crypto_box_seed_keypair(seed):
     __check(sodium.crypto_box_seed_keypair(pk, sk, seed))
     return pk.raw, sk.raw
 
+def crypto_box_beforenm(pk, sk):
+    if None in (pk, sk):
```

Comment:
it is better to use expression pk is None or sk is None.
this helps to avoid of creating temporary tuple and uses operator 'is' instead of operator 'equal'